src/containers/MyAccount/MyAccount.test.helper.ts

```diff
@@ -1,3 +1,4 @@
+import { UPDATE_CURRENT_USER } from '../../graphql/mutations/User';
 import { GET_CURRENT_USER } from '../../graphql/queries/User';
 
 export const MY_ACCOUNT_MOCKS = [
```

Comment:
should we move this file to the `Mocks/` directory?

Reply:
Done